packages/frontend/src/pages/Dashboard.tsx

```diff
@@ -617,103 +618,190 @@ const Dashboard: FC = () => {
-                                config: {
-                                    isDateZoomDisabled,
-                                    pinnedParameters,
-                                },
-                                parameters: dashboardParameters,
-                            });
-                        }}
-                        onCancel={handleCancel}
-                        onMoveToSpace={handleMoveDashboardToSpace}
-                        isMovingDashboardToSpace={isContentActionLoading}
-                        onDuplicate={duplicateModalHandlers.open}
-                        onDelete={deleteModalHandlers.open}
-                        onExport={exportDashboardModalHandlers.open}
-                        setAddingTab={setAddingTab}
-                        onEditClicked={handleEnterEditMode}
-                    />
+                            }}
+                            onCancel={handleCancel}
+                            onMoveToSpace={handleMoveDashboardToSpace}
+                            isMovingDashboardToSpace={isContentActionLoading}
+                            onDuplicate={duplicateModalHandlers.open}
+                            onDelete={deleteModalHandlers.open}
+                            onExport={exportDashboardModalHandlers.open}
+                            setAddingTab={setAddingTab}
+                            onEditClicked={handleEnterEditMode}
+                        />
+                    )
                 }
             >
                 {isDashboardRedesignEnabled ? (
-                    <DashboardHeaderV2
-                        isEditMode={isEditMode}
-                        hasTilesThatSupportFilters={hasTilesThatSupportFilters}
-                        // parameters
-                        parameters={referencedParameters}
-                        parameterValues={parameterValues}
-                        onParameterChange={handleParameterChange}
-                        onParameterClearAll={clearAllParameters}
-                        isParameterLoading={!areAllChartsLoaded}
-                        missingRequiredParameters={missingRequiredParameters}
-                        pinnedParameters={pinnedParameters}
-                        onParameterPin={toggleParameterPin}
-                        // tabs
-                        activeTab={activeTab}
-                        addingTab={addingTab}
-                        dashboardTiles={dashboardTiles}
-                        onAddTiles={handleAddTiles}
-                        onUpdateTiles={handleUpdateTiles}
-                        onDeleteTile={handleDeleteTile}
-                        onBatchDeleteTiles={handleBatchDeleteTiles}
-                        onEditTile={handleEditTiles}
-                        setGridWidth={setGridWidth}
-                        setAddingTab={setAddingTab}
-                    />
+                    <div>
+                        <DashboardHeader
```

Comment:
can this be assigned to a variable to avoid code duplication?

Reply:
sure! it will only be like this during this transition period of v1/v2 living together, but I'll create!